### ISSUE TITLE
[Question] live preview in second window

### ISSUE THREAD

**Issue by sjsadowski** [OXY2DEV/markview.nvim#131]
Apologies if this has been answered elsewhere, if so please link me to the answer - I did not find it when I was looking. I am enjoying this plugin greatly after trying several others, this fits my style/workflow the best. 

Is it possible to live preview with a file? IE after performing a vsplit on the left hand side, markview is disabled, and on the righthand side markview is enabled showing a rendered version of the file. My issue right now is that if I split the window, create a new buffer, and read the file to the buffer, it's now an independent buffer and has no knowledge of the changes made in the original buffer.

So - the reality is that I want markview disabled in the original window, but enabled in the second window. Obviously if both windows are pointed to the same buffer, that won't work - since markview applies to the buffer, not the window.

I didn't see a way to complete this effort currently, but this may be due to my incomplete understanding of windows and buffers. Also it's a trivial matter and I can always just enable/disable as needed so it's not a big deal, just a quality of life thing for me.

So in short:
I want to split/vsplit a window with the same file (possibly the same buffer), but have the rendered version only in one window. Optimally this would also scrollbound to the first window, so as I move through the file the unrendered/rendered versions continue to be side-by-side.

Like I said, if this exists or can be done, just point me in the right direction and give me a few hints!

**ETA:** Might help to have a visual.

<img width="1512" alt="markview_example" src="https://github.com/user-attachments/assets/b002e6cb-7cc3-49c2-b027-5602d132f01e">

**Comment by OXY2DEV** [comment]
https://github.com/user-attachments/assets/d33b843f-10d4-4d44-a4de-95df9907bc72

<sub>Top: Source buffer, Bottom: Preview buffer</sub>

Hm, after testing a bit I think this should be possible though I would still prefer this as a manually actived feature (e.g. via a Command) as otherwise it doesn't quite work when dealing with multiple windows.

**Comment by sjsadowski** [comment]
@OXY2DEV manual would be my preference too! Thanks for even considering it!

**Comment by OXY2DEV** [comment]
You can test out this feature now(only available in the `dev` branch for now).

You can run,
```vim
Markview splitToggle {buffer}
```

> {buffer} is the buffer number, defaults to current buffer.

If you don't encounter any bugs and after I am done fixing any possible issues, I will merge it.
